ports/operationsteel/Operation S.T.E.E.L.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+if [ -f "$controlfolder/device_info.txt" ]; then
```

Comment:
Where did you get these IF statements from?

We're always setting these no matter what.
Only makes the .sh longer